src/pkg/k8s/pods.go

```diff
@@ -142,7 +141,6 @@ func (k *K8s) WaitForPodsAndContainers(target PodLookup, include PodFilter) []co
 
 			}
 
-			k.Log("Ready pods %#v", readyPods)
```

Comment:
I would argue this is useful since it could help show a stuck cluster.

Reply:
What specific information is helpful from this debug message? I would rather debug only the pertinent info (like `name:ns:uuid:image` etc...

Reply:
Knowing the status of the pods is useful when figuring out what Zarf saw as it looked at them - if readyPods is empty or doesn't contain the pods we expect then we know that the pod was likely pending or in some bad state and whatever failure we say isn't less likely due to bad logic.

Reply:
We could be more verbose about what we are seeing by potentially logging more status info in the container section (since we check phase in the else block)